modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -199,6 +199,7 @@ export default class NestedTabs extends Base {
 	onShowTabContent( $requestedContent ) {
 		elementorFrontend.elements.$window.trigger( 'elementor-pro/motion-fx/recalc' );
 		elementorFrontend.elements.$window.trigger( 'elementor/nested-tabs/activate', $requestedContent );
+		elementorFrontend.elements.$window.trigger( 'resize' );
```

Comment:
How resizing the window is related to the video?

Reply:
@davseve

1) You will see `base-tabs.js` also runs `elementorFrontend.elements.$window.trigger( 'resize' );` inside `activateTab()`. We decided to match this functionality in nested-tabs. Certain handlers listen for this event.

2) `background-video.js` listens for the `resize` event to calculate the video size. The problem was that the video width/height was on 0px. So we needed to get `this.changeVideoSize` to run. https://github.com/elementor/elementor/blob/main/assets/dev/js/frontend/handlers/background-video.js#L255

Reply:
See Louis' comment.